contracts/Kasu.sol

```diff
@@ -121,16 +121,27 @@ contract Kasu is KasuStorage, KasuMath {
 
     // [Feature 3] Borrower's dashboard
     // borrower can see all the NFTs they borrowed
-    function viewRentedListings(address borrowerAddress) public view returns (Listing[] memory){
-        uint256[] memory listingIds = _getListingIds();
-        Listing[] memory listings = new Listing[](listingIds.length);
-        for (uint i = 0; i < listingIds.length; i++) {
-            Listing memory listing = _getListingById(listingIds[i]);
-            if (listing.rental.borrowerAddress == borrowerAddress){
-                listings[i] = listing;
+    function viewRentedListings() public view returns (Listing[] memory){
```

Comment:
fyi @amarjandu I'm removing the borrower address here. We probably don't want other users to be able to see another user's rented listings for privacy reasons